src/controllers/movies.js

```diff
@@ -0,0 +1,81 @@
+
+const prisma = require('../utils/prisma')
+
+const getAllMovies = async(req, res) => {
+    return await prisma.movie.findMany({
+        select: {
+            id: true,
+            title: true,
+            runtimeMins: true,
+            createdAt: true,
+            updatedAt: true,
+            screenings: true
+        }
```

Comment:
I was wondering if there's a way to not having to type line 6 to 12 over and over